README.md

```diff
@@ -106,6 +106,11 @@ If one wishes to add a new unit functional to that list, please
 - edit the [prototype file](schema/models_directory/pb/qm/dft/dft_unit_functionals_proto.json) and
 - generate a new list of unit functional by running python tests, for example (via `generate_dft_unit_functionals()` from the [esse.functionals](src/py/esse/functionals.py) python module).
 
+### A word on properties
```

Comment:
This title is a bit too vague (I sense this was following @spectre007's "word on functional"), we aren't writing a novel but technical documentation. Let's change this to more informative: "Generative vs Non-generative keys for Property Schemas". And move `### A word on functionals" down below to "## Additional" section or smth like that.

Reply:
titles changed, and additional sentence added to generative keys description